src/app/_storage/settings.ts

```diff
@@ -233,11 +233,18 @@ export class DateSettings {
       enabled: true,
       abbr: false
     },
+    public ordinalContent = {
+      scaling: 10,
+      offset: 0,
```

Comment:
I understand it makes sense to move the scaling and offset from week to ordinalContent, but this causes a need to change user's settings when updated to accommodate the change, this is something I have done a lot of in the past and has not gone well for some users so I'd like to avoid this as much as possible. So in that case please leave these settings under the week object.

Reply:
That's a really good point; I never thought about how user settings might not carry through!

I moved `offset` and `scaling` back to `week`, but kept 1) frontend UI 2) the `ordinalContent.delimiter` the same. I also added a comment in `settings.ts` indicating that the `offset` and `scaling` controls the entire ordinal contents, but I see that the settings page is overall pretty self-explanatory and doesn't have any comments, so if you'd like to keep it that way, feel free to delete my comments!